### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by alexvonme**

### CODE REVIEW

**Comment by alexvonme** [comment]
[A lot of organizations](https://www.contributor-covenant.org/adopters/) use this one and it seems satisfactory. 

We can add emails later.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the maintainers responsible for enforcement at:
+
+alexvonme at matrix dot org
```
Comment (ccoVeille) [inline CODE_OF_CONDUCT.md:64]: you said we can add email later, but there is one

did you mean change the email ?

**Review by ccoVeille — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (alexvonme) [reply CODE_OF_CONDUCT.md:64]: This is not email. Matrix is a bit more like a decentralization-capable and enryption-capable network. A bit like a slack that you can host and add all sorts of capability to.

We should add emails eventually.

Comment (ccoVeille) [reply CODE_OF_CONDUCT.md:64]: ok

**Pull request merged**